sway/layout.c

```diff
@@ -154,6 +203,30 @@ void move_container(swayc_t *container,swayc_t* root,enum movement_direction dir
 
 }
 
+void update_geometry(swayc_t *container) {
+	if (container->type != C_VIEW) {
+		return;
+	}
+	struct wlc_geometry geometry = {
+		.origin = {
+			.x = container->x + container->gaps / 2,
+			.y = container->y + container->gaps / 2
+		},
+		.size = {
+			.w = container->width - container->gaps,
+			.h = container->height - container->gaps
+		}
+	};
```

Comment:
forgot to add conditional `if is_floating` so it doesnt add gaps when it doesnt need to,
also forgot to reuse this in arrange_windows for the floating windows.